src/styles/blocks/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars > .stars__star {
```

Comment:
I think, it should work without `.stars > `. Just to avoid unnecessary nesting

Reply:
agree, let's remove `.stars >`